networkmanager/networkmanager_test.go

```diff
@@ -18,37 +18,65 @@
 package networkmanager_test
 
 import (
+	"bytes"
+	"context"
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
+	"net"
 	"os"
-	"os/exec"
 	"path"
-	"strconv"
 	"strings"
 	"testing"
 	"time"
 
-	"github.com/aoscloud/aos_common/aoserrors"
-	runtimespec "github.com/opencontainers/runtime-spec/specs-go"
+	cni "github.com/containernetworking/cni/libcni"
+	"github.com/containernetworking/cni/pkg/types"
+	current "github.com/containernetworking/cni/pkg/types/100"
 	log "github.com/sirupsen/logrus"
 
+	"github.com/aoscloud/aos_common/aoserrors"
 	"github.com/aoscloud/aos_servicemanager/config"
 	"github.com/aoscloud/aos_servicemanager/networkmanager"
 )
 
-/*******************************************************************************
- * Vars
- ******************************************************************************/
+/***********************************************************************************************************************
+ * Consts
+ **********************************************************************************************************************/
 
-var (
-	manager *networkmanager.NetworkManager
-	tmpDir  string
-)
+const instanceID = "instance0"
+
+/***********************************************************************************************************************
+ * Types
+ **********************************************************************************************************************/
+
+type cniConfig struct {
+	networkConfig *cni.NetworkConfigList
+}
+
+type cniNetwork struct {
```

Comment:
Can you use cni.NetworkConfigList instead?

Reply:
no, since `cni.ConfListFromBytes` requires a json structure